Library/Formula/sysdig.rb

```diff
@@ -14,18 +11,37 @@ class Sysdig < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "luajit"
+
+  resource "zlib" do
+    url "http://zlib.net/zlib-1.2.8.tar.gz"
+    mirror "https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib-1.2.8.tar.gz"
+    sha256 "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d"
+  end
 
   # More info on https://gist.github.com/juniorz/9986999
   resource "sample_file" do
     url "https://gist.githubusercontent.com/juniorz/9986999/raw/a3556d7e93fa890a157a33f4233efaf8f5e01a6f/sample.scap"
-    sha1 "0aa3c30b954f9fb0d7320d900d3a103ade6b1cec"
+    sha256 "efe287e651a3deea5e87418d39e0fe1e9dc55c6886af4e952468cd64182ee7ef"
   end
 
   def install
     ENV.libcxx if MacOS.version < :mavericks
 
+    resource("zlib").stage do
+      system "./configure", "--prefix=#{buildpath}/zlib"
```

Comment:
Usually, no. `zlib` is typically only required at build time for the times we do need to vendor it. Is there any specific reason it's refusing to use the system version? Does it give any helpful output if we try?